Update Sample5_SignVerify.md

LWChris opened the pull request:
Renamed variables `rsaKey` and `ecKey` to `...KeyOptions`, to better illustrate that the object does not hold the key but the config options that tell the `keyClient` how to create the new key.

# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

LWChris committed:
Update Sample5_SignVerify.md

Renamed variabled `rsaKey` and `ecKey` to `...KeyOptions`, to better illustrate that the object does not hold the key but the config options that tell the `keyClient` how to create the new key.

LWChris commented:
Note: I had a typo in the commit message that I fixed in the PR but I don't think I can fix the commit message any more. It's not a really bad typo though, just "variabled" instead of "variables". But maybe the merging person can fix this in the merge commit.

heaths commented on sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample5_SignVerify.md:
```diff
@@ -20,21 +20,21 @@ First, we'll create both an RSA key and an EC key which will be used to sign and
 };
 
-KeyVaultKey cloudRsaKey = keyClient.CreateRsaKey(rsaKey);
+KeyVaultKey cloudRsaKey = keyClient.CreateRsaKey(rsaKeyOptions);
```
Since you're disambiguating, you might also drop "cloud" as a prefix to this local and `cloudEcKey` below.

LWChris replied:
Wouldn't the "cloud" prefix still be good, since the actual keypair was created in the Azure cloud, and I get a reference to a representation of this cloud-created object?

LWChris replied:
Changed it now, and even removed the "cloud" prefixes. However, when I run the `Update-Snippets.ps1`, none of the md files have changed according to `git status`; I would have expected the prefix-change to be reflected as well?

heaths replied:
They should've, yes. Did you get any errors? Did you pass `-ServiceDirectory keyvault`? Though, even if you didn't do the latter, that would just run over the entirety of the _sdk_ folder.

For expediency and because we appreciate your contribution, I can make the updates inline and get this PR merged.

LWChris replied:
@heaths Yes I used exactly the command as you said, and it did not throw any errors. It said "Replaced <file>" for all md files, including the Sample5_SignVerify.md. I saw the timestamp was touched but it the contents were unchanged. I ran the PS script on ExecutionPolicy Process Unrestricted. Does PowerShell also need an additional permission to allow file content changes (not that I ever had troubles when processing and transforming files with PS scripts)?

heaths reviewed (changes requested):
Thank you for the contribution!

Our samples come from code we run under test, so the changes need to actually be made in _sdk/keyvault/Azure.Security.KeyVault.Keys/test/samples/Sample5_SignVerify.cs_. Then, from the project root, run `eng/scripts/Update-Snippets.ps1 -ServiceDirectory keyvault` to generate the markdown / make sure it's correct (that's why the build validation is failing).

LWChris committed:
Rename variables to better illustrate the actual referenced objects

Rename variable `rsaKey` to `rsaKeyOptions` since that object does not hold the key but the options which tell the keyClient how to create the new RSA key. Rename `cloudRsaKey` to `rsaKey` since that variable contains a reference to a key object. Both changes analogous for EC keys.

heaths committed:
Update Sample5_SignVerify.md
